packages/crayons-core/src/components/data-table/data-table.tsx

```diff
@@ -904,6 +917,40 @@ export class DataTable {
     }
     return closestCell;
   }
+  setResizerHeight() {
+    const resizers = this.el.shadowRoot.querySelectorAll('.resizer');
+    if (resizers.length > 0) {
+      resizers.forEach((item: HTMLElement) => {
+        item.style.height =
+          this.el.shadowRoot.querySelector('table').offsetHeight + 'px';
+      });
+    }
+  }
+
+  resizeCol(e) {
+    const targetCol = e.path[0];
```

Comment:
replace it with composedPath. or do a check to see if e.path exists if not use composedPath

Reply:
fixed